src/contributors page/user_palette.js

```diff
@@ -1,6 +1,6 @@
 const user_palette = [
   {
-    background:"#005b96",
+    background:"green",
     color:"white"
   },
```

Comment:
Hey, why have you changed this! you just needed to add one palette that you have done! revert it back to #005b96